eclipse/mars/gcp-eclipse-mars.tpd

```diff
@@ -18,6 +21,7 @@ location "http://download.eclipse.org/cbi/updates/license" {
 location "http://download.eclipse.org/releases/mars/" {
 	org.eclipse.sdk.feature.group
 	org.eclipse.jdt.feature.group
+  org.eclipse.platform.feature.group
```

Comment:
It didn't seem to be. This was the change that enabled it to find the relevant classes needed for the wizard.